src/components/MoviesList/MoviesList.tsx

```diff
@@ -3,13 +3,26 @@ import React from 'react';
 import { MovieCard } from '../MovieCard';
 
 interface Props {
-  movies: Movie[];
+  movies: Movie[],
+  search: string,
 }
 
-export const MoviesList: React.FC<Props> = ({ movies }) => (
-  <div className="movies">
-    {movies.map(movie => (
-      <MovieCard key={movie.imdbId} movie={movie} />
-    ))}
-  </div>
-);
+export const MoviesList: React.FC<Props> = ({ movies, search }) => {
+  const copy = [...movies];
+  let visibleMovies = [...movies];
```

Comment:
why make copies? does the filter mutate the original array or return the new one?